requirements.txt

```diff
@@ -7,6 +7,7 @@ pyyaml~=5.1.2
 requests~=2.22.0
 requests_toolbelt~=0.9.1
 hydro-serving-grpc==2.1.0
+pandas==1.0.3
```

Comment:
⚠️  I think it'll be better to fix on `major.minor` versions in requirements. Having said that, I don't know the peculiarities of pandas versioning, i.e. do they break public api in patch versions?

```suggestion
pandas~=1.0.3
```

Reply:
Yeah and could you please fix versions of other deps as well? 😀

Reply:
Done